rainbow.py

```diff
@@ -196,11 +193,20 @@ def drive_toward_ball(self, ball):
                 self.found = True
                 print('Close enough, stopping')
             else:
-                print("ball")
                 # follow 0.2, /2 good
-                forward = 0.18
-                turn = float(image_centre_x - x) / image_centre_x / 2.5
+                a_error = AUTO_MAX_AREA - area
+                forward = AREA_P * a_error
+                if self.last_a_error is not 0:
+                    forward -= AREA_D * (self.last_a_error - a_error)
+                t_error  = (image_centre_x - x) / image_centre_x
+                turn = TURN_P * t_error
+                if self.last_t_error is not 0:
```

Comment:
`!=` - This will work, but Python does some crazy identity equality stuff with integers. small ints are fine for `is`-ing, but large ones don't equate in that way. Best not to get in the habit of doing it.